onnxruntime/python/tools/transformers/models/llama/requirements.txt

```diff
@@ -3,4 +3,6 @@ transformers>=4.33.2
 torch>=2.2.0.dev20230920
 onnx>=1.14.0
 datasets>=2.8.0
-protobuf==3.20.2
\ No newline at end of file
+protobuf==3.20.2
+mpi4py
```

Comment:
We should add the extra packages in a new `requirements-70b-model.txt` file. This file's packages are shared across different scenarios. You can see the other `requirements.txt` files for an example.

We should also update the README to note that `requirements-70b-model.txt` is for the 70B model.

Reply:
that is good point, will do